tutorials/smart-contracts/deploy-erc20.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Deploy a ERC20 to Asset Hub
```

Comment:
```suggestion
title: Deploy an ERC-20 to Asset Hub
```

Reply:
Please check the rest of the guide and update to ERC-20 (with hyphen), unless referring to code